### ISSUE TITLE
Certificate issue

### ISSUE THREAD

**Issue by IzzySoft** [fathonyfath/quran-lite-android#5]
A scan ([see here](https://gitlab.com/IzzyOnDroid/repo/-/issues/477) for details and background) just revealed the APKs at your releases are signed using a debug key. As that has security implications, may I ask you to please switch to a proper release key, and provide the corresponding APK signed with it? Thanks in advance!

**Comment by IzzySoft** [comment]
@fathonyfath any chance?

**Comment by fathonyfath** [comment]
I removed all the apks that use debug key. I'll post an update when I have time to check on this. Thanks for the issue submission.

**Comment by IzzySoft** [comment]
Thanks for taking care! Any ETA for a release-key signed APK then (just asking for orientation)?

**Comment by IzzySoft** [comment]
@fathonyfath I'm in the last cleanup run now. Will push your app to the end of the queue a last time as you did respond to work on it. But by the end of this month I cannot postpone it any longer and would have to remove the app from [my repo](https://apt.izzysoft.de/fdroid).

**Comment by IzzySoft** [comment]
Last chance, @fathonyfath – end of month, remaining "debug APKs" will be removed from my repo. Would be a real pity to lose yours.

**Comment by fathonyfath** [comment]
Well noted. Let me catch up with the build tools update first...

**Comment by IzzySoft** [comment]
Thanks! Please try to meet the deadline. I won't argue a day or two, but I cannot keep that issue open much longer. It's almost 2 months now.

**Comment by fathonyfath** [comment]
Hey @IzzySoft , just created new release, together with the notification permission update from Android 13. Should be okay now. Feel free to open new issue if you have more inquiry. Thanks!

**Comment by IzzySoft** [comment]
Great, thanks! That looks perfect now. New release will [go live](https://apt.izzysoft.de/packages/id.thony.android.quranlite/) with the next sync in about 2h:

![image](https://github.com/fathonyfath/quran-lite-android/assets/6781438/daffc6b4-5349-4f9e-9fb1-172bb32e279c)

I only wonder why you've chosen such a long `versionName`, with a timestamp inside even… Wouldn't "1.4.0" suffice? [Embedding build time](https://github.com/fathonyfath/quran-lite-android/blob/main/app/build.gradle#L22) is not such a good idea as it makes it hard to establish [reproducible builds](https://reproducible-builds.org/) (naturally, each build would be different as no two builds will have the same timestamp). Commit hash is hard enough, and you'd not even need to do that yourself as AGP since version 8.3 includes it already itself unless you disabled that.

**Comment by fathonyfath** [comment]
No reason really. I think at the time I started build this I wanted to explore what I could do with gradle build script. I'll keep your suggestion in mind for next release. I'm planning to rewrite the app anyway because the current 'app navigation' (it is not real navigation since it just swap views) is kinda hard to reason with and I made new implementation for view navigation which handle proper navigation.
